examples/isDataVariant/index.rsh

```diff
@@ -0,0 +1,25 @@
+'reach 0.1'
+
+export const DT = Data({A: Null, B: Null});
+
+export const main = Reach.App(() => {
+  const A = Participant('A', {
+    getDT: Fun([], DT),
+  });
+  init();
+
+  A.only(() => {
+    const x = declassify(interact.getDT());
+  });
+  A.publish(x);
+  const isDV = isDataVariant("A", ["A", "B"], x);
+  enforce(isDV);
```

Comment:
This is an immoral use of `enforce` :)

Reply:
Bleh, it's the easiest way to be sure this kind of test code isn't optimized out of a consensus step :p

Reply:
:hankey: